provisions/roles/registry/tasks/main.yml

```diff
@@ -16,16 +16,11 @@
 - name: Run UI registry container.
   sudo: yes
   docker_container:
-      image: registry.centos.org/pipeline-images/docker-registry-frontend:v2
+      image: registry.centos.org/pipeline-images/reg:latest
       name: registry-ui
       state: started
       restart: yes
       restart_policy: always
       pull: yes
-      published_ports: "9000:80"
-      env:
-          ENV_DOCKER_REGISTRY_HOST: "{{ public_registry }}"
-          ENV_DOCKER_REGISTRY_PORT: "5000"
-          ENV_REGISTRY_PROXY_FQDN: "registry.centos.org"
-          ENV_REGISTRY_PROXY_PORT: "443"
-          ENV_MODE_BROWSE_ONLY: "true"
+      published_ports: "9000:8080"
+      command: "--registry registry.centos.org"
```

Comment:
how about replacing the `registry.centos.org` with `{{public_registry}}` ? this will allow the registry to be working with deployment other than prod as well.

Reply:
yeah, you're right. updated!